components/sql/src/main/scala/pl/touk/nussknacker/sql/db/schema/MetaDataProviderFactory.scala

```diff
@@ -13,23 +13,23 @@ object MetaDataProviderFactory {
   private val igniteDriverPrefix = "org.apache.ignite.IgniteJdbc"
 }
 
-class MetaDataProviderFactory {
+class MetaDataProviderFactory extends Serializable {
 
   def create(dbPoolConfig: DBPoolConfig): JdbcMetaDataProvider = {
     val props = new Properties()
     dbPoolConfig.dataSourceProperties.foreach { case (k, v) =>
       props.put(k, v)
     }
-    val ds = ThreadUtils.withThisAsContextClassLoader(getClass.getClassLoader) {
-      new DriverDataSource(
-        dbPoolConfig.url,
-        dbPoolConfig.driverClassName,
-        props,
-        dbPoolConfig.username,
-        dbPoolConfig.password
-      )
-    }
     val getConnection: () => Connection = () => {
+      val ds = ThreadUtils.withThisAsContextClassLoader(getClass.getClassLoader) {
```

Comment:
czy tutaj zrobiłem ok, nie jestem pewien czy to nie będzie mieć negatywnych konsekwencji